extras/kitty/cyberdream.conf

```diff
@@ -21,3 +21,7 @@ color14               #5ef1ff
 color7                #ffffff
 color15               #ffffff
 selection_foreground  #ffffff
+active_tab_foreground   #000
+active_tab_background   #ffbd5e
+inactive_tab_foreground #ffffff
+inactive_tab_background #16181a
```

Comment:
Move these to [/lua/cyberdream/extra/kitty.lua](https://github.com/scottmckendry/cyberdream.nvim/blob/main/lua/cyberdream/extra/kitty.lua)
They will be lost the next time the CI runs otherwise.